travis/show_executor_logs.sh

```diff
@@ -2,17 +2,12 @@
 set -v
 
 echo "what logs are there"
-find $PROJECT_DIR/../scheduler/log -name 'cook*.log'
+find ${TRAVIS_BUILD_DIR}/scheduler/log -name 'cook*.log'
```

Comment:
These two lines look like they should have also been moved to `show_scheduler_logs.sh`, but I'd be in favor of simply deleting them.

Reply:
Done (in PR #674)